dora/core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -220,6 +222,7 @@ static void checkSortConf(AlluxioConfiguration conf) {
    * @throws InvalidPathException if the path is invalid
    * @throws alluxio.exception.AccessControlException if the access is denied
    */
+  // easy
```

Comment:
I used these comment to mark each function with how hard it is to migrate to `UfsUrl`. Let's remove these comments from your PR.